Legion/CourtOfStars/Trash.lua

```diff
@@ -0,0 +1,185 @@
+if not mod then return end
+mod.displayName = CL.trash
+mod:RegisterEnableMob(
+	104246, -- Duskwatch Guard
+	111563, -- Duskwatch Guard
+	104251, -- Duskwatch Sentry
+	104270, -- Guardian Construct
+	104278, -- Felbound Enforcer
+	104277, -- Legion Hound
+	107435, -- Gerenth the Vile & Suspicious Noble
+	104273, -- Jazshariu
+	104275, -- Imacu'tya
+	104274, -- Baalgar the Watchful
+	105715, -- Watchful Inquisitor
+	104295, -- Blazing Imp
+	105705, -- Bound Energy
+	104300, -- Shadow Mistress
+	105704, -- Arcane Manifestation
+	105703, -- Mana Wyrm
+	104247, -- Duskwatch Arcanist
+	105699, -- Mana Saber
+	112668 -- Infernal Imp
+)
+
+--------------------------------------------------------------------------------
+-- Localization
+--
+
+local L = mod:NewLocale("enUS", true)
+if L then
+	L.Guard = "Duskwatch Guard"
+	L.Sentry = "Duskwatch Sentry"
+	L.Construct = "Guardian Construct"
+	L.Enforcer = "Felbound Enforcer"
+	L.Hound = "Legion Hound"
+	L.Gerenth = "Gerenth the Vile"
+	L.Jazshariu = "Jazshariu"
+	L.Imacutya = "Imacutya"
+	L.Baalgar = "Baalgar the Watchful"
+	L.Inquisitor = "Watchful Inquisitor"
+	L.BlazingImp = "Blazing Imp"
+	L.Energy = "Bound Energy"
+	L.Mistress = "Shadow Mistress"
+	L.Manifestation = "Arcane Manifestation"
+	L.Wyrm = "Mana Wyrm"
+	L.Arcanist = "Duskwatch Arcanist"
+	L.Saber = "Mana Saber"
+	L.InfernalImp = "Infernal Imp"
+end
+L = mod:GetLocale()
+
+--------------------------------------------------------------------------------
+-- Initialization
+--
+
+function mod:GetOptions()
+	return {
+		209027, -- Quelling Strike (Duskwatch Guard)
+		209033, -- Fortification (Duskwatch Guard)
+		{209036, "SAY", "FLASH"}, -- Throw Torch (Duskwatch Sentry)
+		225100, -- Charging Station (Guardian Construct)
+		209495, -- Charged Smash (Guardian Construct)
+		209512, -- Disrupting Energy (Guardian Construct)
+		{209413, "SAY", "FLASH"}, -- Suppress (Guardian Construct)
+		211464, -- Fel Detonation (Felbound Enforcer)
+		211391, -- Felblaze Puddle (Legion Hound)
+		214692, -- Shadow Bolt Volley (Gerenth the Vile)
+		214688, -- Carrion Swarm (Gerenth the Vile)
+		{214690, "SAY", "FLASH"}, -- Cripple (Gerenth the Vile)
+		207979, -- Shockwave (Jazshariu)
+		209378, -- Whirling Blades (Imacu'tya)
+		{207980, "SAY", "FLASH"}, -- Disintegration Beam (Baalgar the Watchful)
+		211299, -- Searing Glare (Watchful Inquisitor)
+		212784, -- Eye Storm (Watchful Inquisitor)
+		211401, -- Drifting Embers (Blazing Imp)
+		212031, -- Charged Blast (Bound Energy)
+		{211470, "SAY", "FLASH"}, -- Bewitch (Shadow Mistress)
+		{211473, "SAY", "FLASH"}, -- Shadow Slash (Shadow Mistress)
+		209485, -- Drain Magic (Arcane Manifestation)
+		209477, -- Wild Detonation (Mana Wyrm)
+		209410, -- Nightfall Orb (Duskwatch Arcanist)
+		209404, -- Seal Magic (Duskwatch Arcanist)
+		{209516, "SAY", "FLASH"}, -- Mana Fang (Mana Saber)
+		224377, -- Drifting Embers (Infernal Imp)
+	}, {
+		[209027] = L.Guard,
+		[209033] = L.Guard,
```

Comment:
This repeating isn't necessary. You only need to state each NPC once, it will cover all IDs until the next ID changes NPC, if you get what I mean.